lsp/version.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#define LSP_VER_MAJOR 1
+#define LSP_VER_MINOR 2
+#define LSP_VER_PATCH 0
+
+#define _STR(v) #v
+#define STR(v) _STR(v)
+
+#define STRINGIFY_VERSION(major, minor, patch) STR(major) STR(.) STR(minor) STR(.) STR(patch)
```

Comment:
Use the `LSP_` prefix for all defines. And no leading underscore followed by a capital letter (like in `_STR`) since that is a reserved identifier.